pycaret/classification/oop.py

```diff
@@ -339,21 +339,22 @@ def setup(
         container = []
         container.append(["Session id", self.seed])
         container.append(["Target", self.target_param])
-        container.append(["Target type", "classification"])
+        container.append(["Target type", "Binary" if self.y.nunique() == 2 else "Multiclass"])
```

Comment:
I wonder if we can encapsulate the binary check in a separate function

Reply:
in utils.py? easy enough...